City-of-Helsinki/helsinki-paatokset
# PP-340: Announcement content type and content lists

**To test**
* Checkout branch, run `make drush-cr drush-cim`
* Create a page called "Kuulutukset" at `/tietoa-paatoksenteosta/kuulutukset` if one doesn't exist already
* Create some articles: https://helsinki-paatokset.docker.so/fi/node/add/article
  * Add at least one keyword for each one
  * Test some of the content paragraphs just in case (images, text, etc)
  * Use various years for the publication date
* Check that the layout looks OK and that you see other articles in the sidebar
* Click one of the keywords, this should take you to a term listing page
  * All the articles with the same keyword should be listed on this page
  * You should see the other keywords in the sidebar
* Check the Kuulutukset page
  * This should list all the articles you created and the list should be filterable by year
  * The order should be from the newest to oldest
